templates/azure/private-offer.html

```diff
@@ -1,132 +1,167 @@
 {% extends "azure/base_azure.html" %}
 
 {% block title %}Azure Private Offer{% endblock %}
-{% block meta_description %}Ubuntu on Azure Private Offer - Get exclusive services and support for your large scale Linux operation{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1P4sqhsi6jqWwbqjrBDuAOABGIcbJDM6Efy59ztaroVk/edit{% endblock meta_copydoc %}
 
-{% block head_extra %}
-  <style>
-    h2,
-    .p-heading--2 {
-      font-weight: 300;
-    }
-  </style>
+{% block meta_description %}
+  Ubuntu on Azure Private Offer - Get exclusive services and support for your large scale Linux operation
 {% endblock %}
 
-{% block content %}
-<section class="p-strip--suru-blog-header">
-  <div class="row">
-    <div class="col-7 u-vertically-center">
-      <h1>Get 24/7 SLA support and enhanced security with our private offer</h1>
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1P4sqhsi6jqWwbqjrBDuAOABGIcbJDM6Efy59ztaroVk/edit
+{% endblock meta_copydoc %}
 
-      <p class="u-no-padding--top p-heading--4">Choose the broadest open-source security coverage in the cloud</p>
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
+{% block content %}
+  <section class="p-section--hero">
+    <div class="u-fixed-width">
+      <h1>Get 24/7 SLA support and enhanced security with our private offer</h1>
+      <div class="p-section--shallow">
+        <h2 class="u-no-padding--top">Choose the broadest open-source security coverage in the cloud</h2>
+      </div>
+      <hr class="is-muted" />
       <p>
         <a href="#private-offer-form" class="p-button--positive">Get private offer</a>
       </p>
     </div>
-    <div class="col-5 u-hide--small u-hide--medium u-vertically-center u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/71fe6a11-private-offer-shield.svg",
-        alt="",
-        width="206",
-        height="232",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr />
+      <h2 class="p-text--small-caps">Get Ubuntu Pro for Azure</h2>
+      <div class="p-logo-section">
+        <div class="p-logo-section__items u-no-margin--bottom">
+          <div class="p-logo-section__item">
+            {{ image (
```

Comment:
For all other instances as well pls
```suggestion
            {{ image(
```